lake/branch.go

```diff
@@ -89,6 +94,17 @@ func loadMessage(objects []data.Object) string {
 	return b.String()
 }
 
+func loadMeta(meta string) (zed.Value, error) {
+	if meta == "" {
+		return zed.Value{zed.TypeNull, nil}, nil
+	}
+	zv, err := zson.ParseValue(zed.NewContext(), meta)
+	if err != nil {
+		return zed.Missing, fmt.Errorf("cannot parse zson string %s: %v", zv, err)
```

Comment:
You should probably create a public error called `ErrInvalidCommitMeta` that looks like:
```
var ErrInvalidCommitMeta = errors.New("cannot parse zson string")
```
Then the `handleBranchLoad` handler on line 316 of service/handlers.go should check for this error and return ErrInvalid so the error code will be 400 in the response.

```suggestion
		return zed.Missing, fmt.Errorf("%w %s: %v", ErrInvalidCommitMeta, zv, err)
```

Reply:
Thanks Matt! This makes a lot sense! Done!